modules/ocf_printhost/manifests/cups.pp

```diff
@@ -58,7 +58,16 @@
         content => epp('ocf_printhost/cups/ppd/m806.ppd.epp', { 'double' => true });
     }
   }
-
+  file { '/var/spool/cups/':
```

Comment:
You should modify the mount point a few lines below instead (add `uid=ocfenforcer`)

Reply:
Thanks for the heads up!